packages/nodes-base/nodes/Onfleet/descriptions/TeamDescription.ts

```diff
@@ -83,6 +93,169 @@ const enableSelfAssignmentField = {
+} as INodeProperties;
+
+const serviceTImeField = {
+	displayName: 'Service time',
+	name: 'serviceTIme',
+	type: 'number',
+	default: 2,
+	typeOptions: {
+		minValue: 0,
+	},
+	description: 'The default service time to apply in Minutes to the tasks when no task service time exists.',
+} as INodeProperties;
+
+const routeEndField = {
+	displayName: 'Route end',
+	name: 'routeEnd',
+	type: 'string',
+	default: '',
+	description: 'Where the route will end.',
+} as INodeProperties;
+
+const maxAllowedDelayField = {
+	displayName: 'Max allowed delay',
+	name: 'maxAllowedDelay',
+	type: 'number',
+	default: 10,
+	description: '',
+	typeOptions: {
+		minValue: 1,
+	},
+} as INodeProperties;
+
+const dropoffField = {
+	displayName: 'Dropoff',
+	name: 'dropoff',
+	type: 'boolean',
+	default: false,
+	description: 'Dropoff.',
+} as INodeProperties;
+
+const pickupField = {
+	displayName: 'Pickup',
+	name: 'pickup',
+	type: 'boolean',
+	default: false,
+	description: 'Pickup.',
+} as INodeProperties;
+
+const longitudeDropoffField = {
+	displayName: 'Dropoff longitude',
+	name: 'dropoffLongitude',
+	type: 'number',
+	typeOptions: {
+		numberPrecision: 14,
+	},
+	default: '',
+	description: 'The longitude for dropoff location.',
+} as INodeProperties;
+
+const latitudeDropoffField = {
+	displayName: 'Dropoff latitude',
+	name: 'dropoffLatitude',
+	type: 'number',
+	typeOptions: {
+		numberPrecision: 14,
+	},
+	default: '',
+	description: 'The latitude for dropoff location.',
+} as INodeProperties;
+
+const longitudePickupField = {
+	displayName: 'Pickup longitude',
+	name: 'pickupLongitude',
+	type: 'number',
+	typeOptions: {
+		numberPrecision: 14,
+	},
+	default: '',
+	description: 'The longitude for pickup location.',
+} as INodeProperties;
+
+const latitudePickupField = {
+	displayName: 'Pickup latitude',
+	name: 'pickupLatitude',
+	type: 'number',
+	typeOptions: {
+		numberPrecision: 14,
+	},
+	default: '',
+	description: 'The latitude for pickup location.',
+} as INodeProperties;
+
+const pickupTimeField = {
+	displayName: 'Pickup time',
+	name: 'pickupTime',
+	type: 'dateTime',
+	default: Date.now(),
+	description: 'If the request includes pickupLocation pickupTime must be present if the time is fewer than 3 hours in the future.',
+} as INodeProperties;
+
+const restrictedVehicleTypesField = {
+	displayName: 'Restricted vehicle types',
+	name: 'restrictedVehicleTypes',
+	type: 'options',
+	options: [
+		{
+			name: 'Car',
+			value: 'CAR',
+		},
+		{
+			name: 'Motorcycle',
+			value: 'MOTORCYCLE',
+		},
+		{
+			name: 'Bicycle',
+			value: 'BICYCLE',
+		},
+		{
+			name: 'Truck',
+			value: 'TRUCK',
+		},
+	],
+	default: 'CAR',
+	description: 'Vehicle types to ignore in the query.',
+} as INodeProperties;
+
+const serviceTimeEstimateField = {
+	displayName: 'Service time',
+	name: 'serviceTIme',
+	type: 'number',
+	default: 120,
```

Comment:
I thought the default of this service time is 2 minutes? https://docs.onfleet.com/reference#team-auto-dispatch

Reply:
For the Get driver time estimate endpoint, it's 120 https://docs.onfleet.com/reference#delivery-estimate